.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
No Big Deal but normally you don't add the VS code files. If you want make a `.gitignore` in the root with the following

```
/.vscode
```

and then they'll never get added see https://www.w3schools.com/git/git_ignore.asp?remote=github